ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-tags-input/gio-form-tags-input.component.ts

```diff
@@ -154,6 +177,10 @@ export class GioFormTagsInputComponent implements MatFormFieldControl<Tags>, Con
 
   private _disabled = false;
 
+  public _displayValueWith?: (value: string) => Observable<string>;
```

Comment:
shouldn't we remove the _ if it's a public field ?

Reply:
it's because we have a set displayValueWith. we can not have same prop name